From e2315cdcb7aeeb72f5ca6dedd484f1efa5eb54cd Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Mon, 28 Oct 2024 11:33:46 -0400
Subject: [PATCH] Downstream-only: unpin virtualenv

Upstream wants a old virtualenv to allow an old pip for compatibility
with old celery and httpx, but we must work with what we have.
---
 tox.ini | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tox.ini b/tox.ini
index 75d74dbb..c18b63b7 100644
--- a/tox.ini
+++ b/tox.ini
@@ -15,7 +15,7 @@
 [tox]
 requires =
     # This version introduced using pip 24.1 which does not work with older Celery and HTTPX versions.
-    virtualenv<20.26.3
+    virtualenv
 envlist =
     # === Common ===
     {py3.6,py3.7,py3.8,py3.9,py3.10,py3.11,py3.12,py3.13,py3.14,py3.14t}-common
-- 
2.47.0
